data/filters/templates/youtube-cleanup.yaml

```
@@ -66,6 +70,9 @@ template: |
   {{! Applied to both www. and m. sites, defuses custom pan events for swipe }}
   youtube.com##+js(aeld, /^pan(start|move|end)$/)
   {{/if}}
+  {{#if no-ambient-mode}}
+  www.youtube.com###cinematics.ytd-watch-flexy
```

From my benchmarking with Firefox on macOS, your rule does only negate half of the CPU overhead of the ambient mode: the gradient does not need to be displayed, but it is still updated by javascript code.

Using `:remove()` seems like it completely nullifies the CPU cost, probably by making the javascript loop fail to start. Interestingly, it even removes the parameter item.

```suggestion
  www.youtube.com###cinematics.ytd-watch-flexy:remove()
```